extensions/dicom-html/src/index.js

```diff
@@ -3,7 +3,7 @@ import OHIFDicomHtmlSopClassHandler from './OHIFDicomHtmlSopClassHandler.js';
 
 const OHIFDicomHtmlViewport = asyncComponent(() =>
   import(
-    /* webpackChunkName: "OHIFDicomHtmlViewport" */ './OHIFDicomHtmlViewport.js'
+    /* webpackChunkName: "ConnectedOHIFDicomPDFViewport" */ './OHIFDicomHtmlViewport.js'
```

Comment:
This shouldn't have been changed, it's in the wrong extension.

Reply:
I was removing those comment from all imports but I have added them back, missed this one. Ill update asap, thanks!

Reply:
Done.